### PR TITLE
improved driving for HMD

### PR DESCRIPTION
**Pull request by AndrewMeadows**
- improved collision behavior for HMD when driving

### CODE REVIEW

File: interface/src/avatar/MyCharacterController.cpp [hunk 1]
```diff
@@ -236,109 +283,64 @@ glm::vec3 MyCharacterController::computeHMDStep(const glm::vec3& position, const
...
+        rayResult.m_collisionObject = nullptr;
+
+        if (_ghost.rayTest(rayStart, rayEnd, rayResult)) {
+            btScalar adjustedHitFraction = (rayResult.m_closestHitFraction * expandedStepLength - MIN_STEP_MARGIN) / (stepLength + MIN_FORWARD_SLOP);
+            if (adjustedHitFraction < 1.0f) {
+                steppingUp = true;
+                break;
             }
         }
     }
```
Comment (AndrewMeadows) [inline interface/src/avatar/MyCharacterController.cpp]: The third scan is to figure out if the character should be stepping up over a low obstacle.  We don't yet take full advantage of this info but we may use it more in the future.

File: interface/src/avatar/MyCharacterController.cpp [hunk 2]
```diff
@@ -405,23 +407,23 @@ void MyCharacterController::initRayShotgun(const btCollisionWorld* world) {
     const btScalar topHeight = fullHalfHeight - divisionLine;
     const btScalar slop = 0.02f;
 
-    const int32_t numRows = 3; // must be odd number > 1
-    const int32_t numColumns = 3; // must be odd number > 1
+    const int32_t NUM_ROWS = 5; // must be odd number > 1
+    const int32_t NUM_COLUMNS = 5; // must be odd number > 1
```
Comment (AndrewMeadows) [inline interface/src/avatar/MyCharacterController.cpp]: My performance tests showed that the rayTests are fast and inexpensive so I increased the number of points.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 70bfaf5298776f1beecc1bbd293d9af7bbcf4d8b available at: https://gist.github.com/3b47bb6fca094805f4772f5eb1b512fe<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8926-70bfaf5.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8926-70bfaf5.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 5e273664dbd19998c106a4b72b46896d133d73ec available at: https://gist.github.com/3b47bb6fca094805f4772f5eb1b512fe<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8926-5e27366.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR8926-5e27366.dmg

**Pull request merged**